CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# This CODEOWNERS file denotes the project leads
+# and encodes their responsibilities for code review.
+
+# Instructions: At a minimum, replace the '@GITHUB_USER_NAME_GOES_HERE'
+# here with at least one project lead.
+
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# The format is described: https://github.blog/2017-07-06-introducing-code-owners/
+
+# These owners will be the default owners for everything in the repo.
+*       @decentralgabe 
```

Comment:
can you add Andrés and Mic please?

Reply:
Go for it! PR's here for your convenience but the project is yours to run/admin